### PR TITLE
Verbiage changes mar2022

### PR DESCRIPTION
**Pull request by b-f-chan**
Verbiage changes to multiple pages requested in March 2022

### CODE REVIEW

**Comment by b-f-chan** [comment]
@caravinci if you don't mind doing the PR review while @jaserud is out sick, thank you! 🙏

File: components/pages/about/CovidCloudPane.tsx [hunk 1]
```diff
@@ -46,20 +46,20 @@ const CovidCloudPane = (): ReactElement => {
         width: calc(100% - 100px);
       `}
     >
-      <img src="/images/covid-cloud.png" alt="logo for Covid Cloud" width="180" />
+      <img src="/images/viral-ai.png" alt="logo for Viral AI" width="180" />
```
Comment (caravinci) [inline components/pages/about/CovidCloudPane.tsx:49]: I see a reference here to a file that doesn't currently exist in the repo, and is not being added with the PR.
Going to assume it's a mistake?

**Review by caravinci — CHANGES_REQUESTED** [review]
Changes look good to me. There's reference to a missing image, and I'm pending confirmation on how we want to proceed, before approving the PR

File: components/pages/about/CovidCloudPane.tsx (hunk 1 above)
Comment (b-f-chan) [reply components/pages/about/CovidCloudPane.tsx:49]: Hmm weird I think the file was added in my previous commits/pushes as we discussed yesterday?  I thought the file would get pushed with my branch

File: components/pages/team/Cloud.tsx [hunk 2]
```diff
@@ -81,13 +83,13 @@ const Cloud = (): ReactElement => (
...
         <StyledLink
-          href="https://virusseq.covidcloud.ca/"
+          href="https://viral.ai/collections/virusseq/overview"
           rel="noopener noreferrer"
           target="_blank"
         >
           <img
-            src="images/covid-cloud.png"
-            alt="Covid Cloud Logo"
+            src="images/viral-ai.png"
```
[the remark this answers is not in the document]
Comment (b-f-chan) [reply components/pages/team/Cloud.tsx:91]: Same comment/question as above - Do I need to manually upload this to my branch?

File: components/pages/about/CovidCloudPane.tsx [hunk 3]
```diff
@@ -46,20 +46,20 @@ const CovidCloudPane = (): ReactElement => {
...
-        Genome Canada has also partnered with{' '}
-        <StyledLink href="https://www.dnastack.com" rel="noopener noreferrer" target="_blank">
-          DNAstack
-        </StyledLink>{' '}
-        to integrate VirusSeq data with their COVID Cloud Platform. COVID Cloud is a cloud-based
-        platform that helps researchers find, visualize, and analyze genomics and other datasets
-        related to COVID-19.
+        Genome Canada has also partnered with DNAstack to share{' '}
+        <StyledLink href="https://viral.ai/collections/virusseq/overview" rel="noopener noreferrer" target="_blank">
+          VirusSeq data on Viral AI
```
[the remark this answers is not in the document]
Comment (b-f-chan) [reply components/pages/about/CovidCloudPane.tsx:58]: Very bad marketing lol

File: components/pages/about/CovidCloudPane.tsx (hunk 1 above)
Comment (b-f-chan) [reply components/pages/about/CovidCloudPane.tsx:49]: File uploaded to this branch, please re-check!

**Pull request merged**